agendas/2018-02-01.md

```diff
@@ -44,6 +44,8 @@ Mike Marcacci        | Boltline      | San Diego, CA
 1. Determine volunteers for note taking (5m)
 1. Discuss upcoming meeting schedule (5m)
 1. Present [graphql-import](https://github.com/graphcool/graphql-import) (5m)
+1. Discuss exposing schema metadata (30m)
```

Comment:
Great, thanks a lot for adding this agenda topic, @xuorig! 👍 I was also thinking about adding it, should be an interesting discussion.